src/styles/nav.scss

```diff
@@ -0,0 +1,109 @@
+@import "./stars.scss";
+
+body {
+  margin: 0;
+}
+
+.catalog {
```

Comment:
create separate files for the catalog and the card. why the file with this blocks call nav?